packages/next/build/swc/.cargo/config

```diff
@@ -0,0 +1,4 @@
+[build]
```

Comment:
This is temporary. I'll remove this after implementing support for stable rust in testing system of swc.